docs/src/data/test/props-table.test.ts

```diff
@@ -0,0 +1,6 @@
+import * as fs from 'fs';
+
+test('Props Table', () => {
```

Comment:
This file shouldn't be added: https://github.com/aws-amplify/amplify-ui/blob/main/docs/tests/props-table.test.ts